NitroxModel/Discovery/Models/Platform.cs

```diff
@@ -1,25 +1,24 @@
-﻿using System.ComponentModel;
+using System.ComponentModel;
 
-namespace NitroxModel.Discovery
+namespace NitroxModel.Discovery.Models;
+
+public enum Platform
 {
-    public enum Platform
-    {
-        [Description("Standalone")]
-        NONE,
+    [Description("Unknown")]
+    NONE = 0,
 
-        [Description("Pirated")]
-        PIRATED,
+    [Description("Steam")]
+    STEAM = 1 << 0,
 
-        [Description("Epic Games Store")]
-        EPIC,
+    [Description("Epic Games Store")]
+    EPIC = 1 << 1,
 
-        [Description("Steam")]
-        STEAM,
+    [Description("Microsoft")]
+    MICROSOFT = 1 << 2,
 
-        [Description("Microsoft")]
-        MICROSOFT,
+    [Description("Discord")]
+    DISCORD = 1 << 3,
 
-        [Description("Discord")]
-        DISCORD
-    }
+    [Description("Pirated")]
+    PIRATED = 255
```

Comment:
enum is `int` by default. 

This enum is supposed to contain real game stores. I recommend just removing the "Pirated" flag, and infer it from the detection trigger instead.